constants/fokabotCommands.py

```diff
@@ -165,24 +175,40 @@ async def moderated(fro: str, channel_name: str, message: list[str]) -> str:
         # Turn on/off moderated mode
         # NOTE: this will raise exceptions.channelUnknownException if the channel doesn't exist
         await channelList.updateChannel(channel_name, moderated=enable)
-
-        return f'This channel is {"now" if enable else "no longer"} in moderated mode!'
+        userID = userUtils.getID(fro)
+        await rap_logs.send_rap_log(
+            userID,
+            f"has toggled moderated mode in {channel_name}.",
+        )
+        await rap_logs.send_rap_log_as_discord_webhook(
+            message=f"[{fro}](https://akatsuki.gg/u/{userID}) ({userID}) has toggled moderated mode in {channel_name}.",
+            discord_channel="ac_general",
+        )
+        response = (
+            f'This channel is {"now" if enable else "no longer"} in moderated mode!'
+        )
     except exceptions.channelUnknownException:
-        return "Channel doesn't exist. (??? contact cmyui(#0425))"
+        response = "Channel doesn't exist. (??? contact cmyui/tsunyoku)"
```

Comment:
this actually isn't weird, idk why i had this here
```suggestion
        response = "Channel doesn't exist."
```

Reply:
yep removed that in b346c9ca6da9aa4aa864d14e140ae451cf67bbba